src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/integration/AuthorizationEnabledIntegrationTest.java

```diff
@@ -98,7 +98,7 @@ public void setUpWireMock() {
                                 + "  \"forename\": \"Super\","
                                 + "  \"surname\": \"User\","
                                 + "  \"email\": \"super.user@hmcts.net\","
-                                + "  \"locked\": \"false\","

```

Comment:
Could you please add 1 more integration test case for Suspended status ? (i.e. active =false and pending=false). Need to add one more mock for this.

Reply:
Added this test in latest commit 👍